src/test/java/pages/HomePage.java

```diff
@@ -1,20 +1,67 @@
 package pages;
 
 import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
 
-public class HomePage extends BasePage {
+import java.util.List;
 
+public class HomePage extends BasePage {
     By userAvatarIcon = By.cssSelector("img.avatar");
+    By allSongs = By.xpath("//a[contains(text(),'All Songs')]");
+    By newPlaylistIcon = By.xpath("//i[@class='fa fa-plus-circle create']");
+    By newPlaylistOption = By.xpath("//li[contains(text(), 'New Playlist')]");
+    By newPlaylistField = By.xpath("//input[@name='name']");
+
+    By myFirstPlaylist = By.cssSelector(".playlist:nth-child(3)");
+    By deletePlaylistOption = By.xpath("//li[text()='Delete']");
+    By notificationPopUp = By.xpath("//div[@class='success show']");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.


I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```